app/src/main/java/com/humara/nagar/utils/Utils.kt

```diff
@@ -1,14 +1,52 @@
 package com.humara.nagar.utils
 
+import android.content.Context
+import android.location.Address
+import android.location.Geocoder
+import android.location.LocationManager
 import com.humara.nagar.constants.Constants
+import java.util.*
 
 /**
  * Utils class
  */
 class Utils {
     companion object {
+
         fun getMobileNumberWithCountryCode(mobileNumber: String): String {
             return Constants.COUNTRY_CODE.plus(" $mobileNumber")
         }
+
+        fun getLongitudeAndLatitude(context: Context) : List<Double> {
+            val locationManager = context.getSystemService(Context.LOCATION_SERVICE) as LocationManager
+            val lastKnownLocation = locationManager.getLastKnownLocation(LocationManager.GPS_PROVIDER)
+            if (lastKnownLocation != null) {
+                val longitude = lastKnownLocation.longitude
+                val latitude = lastKnownLocation.latitude
+                return listOf(longitude, latitude)
+            }
+            return emptyList()
+        }
+
+        fun getAddressFromLongAndLat(
+            context: Context,
+        ) : List<Address> {
+
+            val longitudeAndLatitude = getLongitudeAndLatitude(context)
+            if (longitudeAndLatitude.isNotEmpty()) {
+
+                val geocoder = Geocoder(context, Locale.getDefault())
+                val addresses = geocoder.getFromLocation(
```

Comment:
un-checked exception handling. Got this crash while picking location
<img width="1678" alt="Screenshot 2023-02-11 at 1 27 11 PM" src="https://user-images.githubusercontent.com/26206071/218247277-8109b47e-adc8-412e-97f7-971a9bea0ca6.png">

Reply:
Testing this.

Reply:
Fixed. (Added try-catch)